aspnetcore/performance/response-compression.md

```diff
@@ -1,97 +1,114 @@
 ---
-title: Response Compression Middleware
+title: Response Compression Middleware for ASP.NET Core
 author: guardrex
-description: An introduction to response compression with instructions on how to use Response Compression Middleware in ASP.NET Core applications.
-keywords: ASP.NET Core, performance, response compression, gzip, accept-encoding, middleware
+description: Learn about response compression and how to use Response Compression Middleware in ASP.NET Core apps.
+keywords: ASP.NET Core,performance,response compression,gzip,accept-encoding,middleware
 ms.author: riande
 manager: wpickett
-ms.date: 03/13/2017
+ms.date: 08/20/2017
 ms.topic: article
 ms.assetid: de621887-c5c9-4ac8-9efd-f5cc0457a134
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: performance/response-compression
 ---
-# Response Compression Middleware
+# Response Compression Middleware for ASP.NET Core
 
 By [Luke Latham](https://github.com/GuardRex)
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample)
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/samples)
 
-Network bandwidth is a limited resource. If you can reduce response payload sizes and thus send less data to clients, you can usually increase the responsiveness of your application, sometimes dramatically. One way to reduce payload sizes is to compress an application's responses.
+Network bandwidth is a limited resource. If you can reduce response payload sizes and thus send less data to clients, you can usually increase the responsiveness of your app, often dramatically. One way to reduce payload sizes is to compress an app's responses.
```

Comment:
> Reducing response size ...

... seems a little awkward just b/c using "response" as an adjective seems harder to grok. Let's try on the next commit with ...

> Reducing the size of the response ...